### PR TITLE
fix(board): Board style reverted to card-pf from flex

### PR DESCRIPTION
**Pull request by sanbornsen**

### CODE REVIEW

**Comment by sanbornsen** [comment]
@mindreeper2420 did I miss it any other places? 

It looks like 
screen shot 2017-04-20 at 12 54 23 am

**Comment by fabric8cd** [comment]
@sanbornsen snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1692-fabric8-planner.int.rdu2c.fabric8.io

File: src/app/work-item/work-item-board/work-item-board.component.html [hunk 1]
```diff
@@ -38,8 +38,9 @@
                     </div>
                   </div>
                   <div [dragula]='"first-bag"' class="card-wrapper">
-                    <div class="board-card flex-container in-column-direction flex-grow-1" *ngFor='let item of lane.workItems' [attr.data-id]="item.id"
-                         (mousedown)="getWI(item)" (touchstart)="getWI(item)">
+                    <div class="card-pf card-pf-view card-pf-view-select card-pf-view-single-select"
```
Comment (mindreeper2420) [inline src/app/work-item/work-item-board/work-item-board.component.html]: @sanbornsen you need to add the class `board-card` back to have the card content align properly

with `board-card`:
screen shot 2017-04-19 at 3 43 21 pm

**Comment by mindreeper2420** [comment]
also, not sure if it is because of this PR or another, but the Work Item titles and Iteration title are now blue

**Comment by sanbornsen** [comment]
@mindreeper2420 thanks. Fixed that. 
Yesh those titles are links in this PR. I was about to ask help from @SMahil regarding that. I was unable to put anchor on whole card (it was breaking the design), so just put the anchor on the title. Same with iteration panel and list item in list page. I need help on that.

**Comment by fabric8cd** [comment]
@sanbornsen snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1692-fabric8-planner.int.rdu2c.fabric8.io

**Comment by mindreeper2420** [comment]
@sanbornsen as an example, can the ` [routerLink]="['./detail/' + item.id]" [queryParams]="existingQueryParams" ` be on the `<span>` or a `<p>` or `<h#>` and still work properly? It would be quite a CSS complication to have to restyle all of the `<a>` in the Board View, List View and Iteration Panel to not be link styling.

**Comment by sanbornsen** [comment]
@mindreeper2420 true, but then if you try to do right click on that and copy the URL or open it on a new tab, it'll not work. If that is okay then we are good.

**Comment by sanbornsen** [comment]
@mindreeper2420 let's take the discussion over MM.

**Comment by fabric8cd** [comment]
@sanbornsen snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1692-fabric8-planner.int.rdu2c.fabric8.io

**Comment by mindreeper2420** [comment]
I would say that this PR is fine and that another should be opened for the CSS changes of the link items.

**Pull request merged**